src/assets/scripts/client/ui/UiController.js

```diff
@@ -888,12 +888,12 @@ class UiController {
      * @param event {jquery event}
      */
     onToggleTraffic(event) {
+        this.$toggleTraffic.toggleClass(SELECTORS.CLASSNAMES.ACTIVE);
         EventTracker.recordEvent(
             TRACKABLE_EVENT.OPTIONS,
             'traffic',
             `${!this.$toggleTraffic.hasClass(SELECTORS.CLASSNAMES.ACTIVE)}`
```

Comment:
Huh, this one had a NOT on the boolean. Fixed in https://github.com/openscope/openscope/pull/1740/commits/16237013cdf530ac399a7b271128315c096eaa9a.